contact/form.html

```diff
@@ -14,24 +14,22 @@
         </p>
     </div>
     <div class="col-md-6">
-        <!-- send this to another host as GitHub Pages doesn't support mail() -->
-        <form role="form" action="http://ccgi.jakeman.plus.com/or/forward_message.php" method="get">
+        <!-- send this to another host (formspark.io) as GitHub Pages doesn't support -->
+        <form action="https://submit-form.com/kj7fWXmt">	
           <div class="form-group">
             <label for="emailAddress">Email address</label>
-            <input type="email" class="form-control" id="emailAddress" name="from" placeholder="Enter your email address. (We do not share this.)" autofocus required>
+            <input required name="sender" type="email" class="form-control" placeholder="Enter your email address. (We do not share this.)" autofocus />
```

Comment:
```suggestion
            <input required name="sender" type="email" class="form-control" placeholder="Enter your email address. (We do not share this.)" autofocus>
```
Minor style thing: The trailing slash is not required by HTML5 and I personally don't use it.